src/core_plugins/elasticsearch/lib/create_admin_cluster.js

```diff
@@ -0,0 +1,19 @@
+export default function (server) {
+  const config = server.config();
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
+
+  class AdminClientLogging extends Logger {
```

Comment:
we have to keep exports at the top level though, and this extends a property coming off the server object, which is passed in